scripts/tvm_cli/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04
+
+COPY install_tvm.sh /tmp/install_tvm.sh
+RUN /tmp/install_tvm.sh
+
+RUN pip3 install --upgrade pip && pip3 install onnx tensorflow pyyaml jinja2
```

Comment:
When running this command in the Docker build, I get the following error:

```
ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

google-auth 1.22.1 requires setuptools>=40.3.0, but you'll have setuptools 39.0.1 which is incompatible.
tensorboard 2.3.0 requires setuptools>=41.0.0, but you'll have setuptools 39.0.1 which is incompatible.
tensorflow 2.3.1 requires numpy<1.19.0,>=1.16.0, but you'll have numpy 1.19.2 which is incompatible.
```

It does not stop the build but I'm worried about the deprecation warning causing errors very soon. Thoughts?

Reply:
Additionally, the `setuptools` errors look problematic too.

Reply:
Thanks for pointing these out, I was able to reproduce the first issue but I'm not getting any error/warning regarding setuptools. Was the setuptools error coming from the setuptools installation?

Reply:
@lucafos
I think @JWhitleyWork meant
```
google-auth 1.22.1 requires setuptools>=40.3.0, but you'll have setuptools 39.0.1 which is incompatible.
tensorboard 2.3.0 requires setuptools>=41.0.0, but you'll have setuptools 39.0.1 which is incompatible.
tensorflow 2.3.1 requires numpy<1.19.0,>=1.16.0, but you'll have numpy 1.19.2 which is incompatible.
```

Reply:
@LucaFos, you may need to update setuptools to the latest version before pip install anything.

Reply:
Just checked, the setuptools issue got solved together with the deprecation warning.

Reply:
Verified as resolved. Thanks!